### ISSUE TITLE
version and os_version in JSON report.json are wrong/imprecise for Edge

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt#15156]
With the help of @mustjab and @thejohnjansen I have been experimenting with Edge runs on Windows 10 in Azure Pipelines on a branch in this repo. Here is the first successful full run, which excluded WebDriver (wdspec) tests:
https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=5330

https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=5267 was an earlier partial run where I used the `edge` product, which is using Selenium.

In both cases, the `run_info` in wpt_report.json looked like this for `version` and `os_version`:
```
    "run_info": {
        "bits": 64,
        "debug": false,
        "has_sandbox": true,
        "os": "win",
        "os_version": "10.0",
        "processor": "x86_64",
        "product": "edge",
        "revision": "c646d5d629d22d8e2448991e02863eebea74e75f",
        "service_pack": "",
        "verify": false,
        "version": "10.0.17763",
        "wasm": false
    },
```

10.0.17763 should be the `os_version`, and `version` is just wrong.

**Comment by foolip** [comment]
@gsnedders has the answer in https://stackoverflow.com/a/52056711

**Comment by foolip** [comment]
On the Windows 10 VMs now in the agent pool, that commands returns 44.17763.1.0.

In https://storage.googleapis.com/wptd-results/9d21b523b4959f402769d58248e4587fda7f39ff/edge-17-windows-10-1a6165dd70/report.json, the latest Edge run from Buildbot, the version is simply "17". The UA string for  Edge 44.17763.1.0 ends with "Edge/18.17763". That's actually the EdgeHTML version, but for continuity with previous builds and the way the rest of the world talks about Edge versions, I think "18.17763" would be the best thing to put in the reports here. Or perhaps just "18" since we also have the detailed OS version.

**Comment by foolip** [comment]
Sent https://github.com/web-platform-tests/wpt/pull/15172 to get the `browser_version` field. When done inside of wpt I'll make no attempt to fix the version to be 18.

**Comment by gsnedders** [comment]
Fixed by #15172.
